third_party/cuda/BUILD

```diff
@@ -19,7 +19,7 @@ cc_library(
     name = "cudart",
     srcs = select({
         ":aarch64_linux": [
-            "targets/aarch64-linux/lib/libcudart.so",
+            "targets/aarch64-linux-gnu/lib/libcudart.so",
```

Comment:
This is the wrong path on stock JetPack

Reply:
Jetpack 4.4 seems to fail without this suffix. Should we tag this release and then change for future JP releases?

Reply:
Fixed it